tests/no-rpath-pie-powerpc.sh

```diff
@@ -37,9 +37,9 @@ if [ "$(echo "$readelfData" | grep -c "PHDR")" != 1 ]; then
 fi
 
 virtAddr=$(echo "$readelfData" | grep "PHDR" | awk '{print $3}')
-if [ "$virtAddr" != "0x00000034" ]; then
+if [ "$virtAddr" != "0x01040000" ]; then
```

Comment:
Previously PHDR was always kept right after EHDR, while now it's allocated at the end of the file.